internal/storage/hub/bundle.go

```diff
@@ -214,9 +214,27 @@ func (b *Bundle) GetFirstMatch(_ context.Context, candidates []namer.ModuleID) (
 	return nil, nil
 }
 
-// GetAll attempts to retrieve all policies from the passed modIDs, unlike `GetFirstMatch` which returns the first
+func (b *Bundle) GetAll(ctx context.Context) ([]*runtimev1.RunnablePolicySet, error) {
+	res := []*runtimev1.RunnablePolicySet{}
+	for fqn := range b.manifest.PolicyIndex {
+		modID := namer.GenModuleIDFromFQN(fqn)
+
+		policySet, err := b.GetFirstMatch(ctx, []namer.ModuleID{modID})
```

Comment:
We could extract the loop body of `GetFirstMatch` (minus the `stat`'ing) into its own function and call that from the `GetAll*` methods. (It feels a bit weird to call `GetFirstMatch` from here.  Furthermore, since we are looping through the manifest, the files are guaranteed to exist as well.)